scrapd/core/apd.py

```diff
@@ -179,14 +179,42 @@ def parse_twitter_description(twitter_description):
     return sanitize_fatality_entity(d)
 
 
+def parse_detail_page_description(details_description):
+    """
+    Clean up a details page notes section.
+
+    The purpose of this function is to attempt to extract the sentences about
+    the crash with some level of fidelity, but does not always return
+    a perfectly parsed sentence as the html syntax varies widely.
+
+    :param str details_description: the paragraph after the Deceased information
+    :return: A paragraph containing the details of the fatality in sentence form.
+    :rtype: str
+    """
+    start_tag = details_description.find('<p>') + 3
+    end_tag = details_description.find('</p>', start_tag)
+    if not details_description[start_tag:end_tag].upper().isupper():
```

Comment:
Actually, I think .upper().isupper() will return False if there are no letters in the string. I tested for strings of whitespace and with numbers only.

Reply:
If there's a better way of testing for strings that contain no letters, though, I'm happy to use that.

Reply:
Ah I see. Maybe something with `not .isalpha()` would work? Otherwise don't sweat it too much, simply put a comment above explaining why you did that and you're good to go.